src/core/openthread.cpp

```diff
@@ -96,6 +96,19 @@ const uint8_t *otGetExtendedAddress(void)
     return reinterpret_cast<const uint8_t *>(sThreadNetif->GetMac().GetExtAddress());
 }
 
+ThreadError otSetExtendedAddress(const otExtAddress *aExtAddress)
+{
+    ThreadError error = kThreadError_None;
```

Comment:
I think it makes sense to enforce the i/g bit.  I'm not sure we want to enforce the u/l bit yet.  The client may want to clear the u/l bit for some reason.